code/modules/paperwork/paper.dm

```diff
@@ -10,6 +10,13 @@
 #define MODE_READING 0
 #define MODE_WRITING 1
 #define MODE_STAMPING 2
+#define CATEGORY_COMMAND "Command Department"
+#define CATEGORY_SECURITY "Security Department"
+#define CATEGORY_ENGINEERING "Engineering Department"
+#define CATEGORY_RESEARCH "Research Department"
+#define CATEGORY_MEDICAL "Medical Department"
+#define CATEGORY_SUPPLY "Supply Department"
+#define CATEGORY_OTHER "Other"
```

Comment:
move all of these into a new file in the _DEFINES folder

Reply:
p.s. i'm saying this because there are 13 of them at this point, some defines in paperbin, some defines in photocopier, and so on.